code/game/gamemodes/blob/blobs/blob_mobs.dm

```diff
@@ -30,6 +30,8 @@
 			var/obj/effect/temp_visual/heal/H = new /obj/effect/temp_visual/heal(get_turf(src)) //hello yes you are being healed
 			if(overmind)
 				H.color = overmind.blob_reagent_datum.complementary_color
+				if(H.color == "#a15656") //the colour for ripping tendrils
+					H.color = "#5995BA" //bye red cross
```

Comment:
these should be defines. if someone changes the colour of the blob slightly this code no longer works.

Reply:
yes I want to keep the colour changing intact for the other blob mutations. It's only ripping tendrils that is the issue since it's red. The only time it wouldn't be #a15656 and instead a different shade of red is if an admin changed it on purpose

Reply:
I don't mean changed by an admin, i mean changed by a future PR.

Reply:
should I define all blob colours under like colors.dm then? it'd be weird only to do ripping tendrils

Reply:
Please do.